src/Command/ErrorsConsoleStyle.php

```diff
@@ -118,6 +118,7 @@ private function wrap(array $rows, int $terminalWidth, int $maxHeaderWidth): arr
 	public function createProgressBar(int $max = 0): ProgressBar
 	{
 		$this->progressBar = parent::createProgressBar($max);
+		$this->progressBar->setFormat(' %current%/%max% [%bar%] %percent:3s%% %elapsed:6s%/%estimated:-6s%');
```

Comment:
Shows as 
```
 1486/1486 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100% 56 secs/56 secs
```

I copied original debug verbosity format and removed memory info to avoid confusion. Now everybody will see it this way no matter the verbosity level is selected. I think it is acceptable.

Reply:
Doesn't this mean that the progress bar will be more verbose even without `-vvv`?

Reply:
Fixed, it should behave as it was, just not showing the memory in debug mode.